DEVELOPMENT.md

```diff
@@ -4,20 +4,24 @@
 ## Contents
 
 - [First-time Setup](#first-time-setup)
-   - [Go Language Installation (if absent)](#1-go-language-installation-if-absent)
-   - [Verification of Go Installation](#2-verification-of-go-installation)
-   - [Project Execution](#3-project-execution)
-   - [Build the Project](#4-build-the-project)
+  - [Go language installation (if absent)](#go-language-installation-if-absent)
+  - [Verification of Go installation](#verification-of-go-installation)
+- [Developing the Bechamel API](#developing-the-bechamel-api)
```

Comment:
change at line 8.5,
part 1 of 2 - add:
```markdown
   - [Docker Installation and Usage (if absent)](#docker-installation-and-usage-if-absent)
```

Reply:
@patmejia aiming to add Docker installation instructions to a more general document outside this repo, since it'll be relevant to project-ricotta and bechamel-api developers both. Hopefully coming soon!